Lib/fontmake/font_project.py

```diff
@@ -36,6 +36,7 @@
 from ufo2ft.featureWriters import FEATURE_WRITERS_KEY, loadFeatureWriters
 from ufo2ft.util import makeOfficialGlyphOrder
 
+import fontmake.instantiator
```

Comment:
i like `from fontmake import instantiator` better; `fontmake.instantiator` is too long